src/MonitoringFunctions.Windows/Functions/WorkItemCreator.cs

```diff
@@ -101,6 +124,18 @@ internal static class WorkItemCreator
 
                 log.LogInformation(successMessage);
                 createdWorkItems.Add(workItem);
+
+                if (teamsConnector != null)
+                {
+                    try
+                    {
+                        await teamsConnector.SendIncidentCard("There is a new issue with the install scripts.", title, workItemUrl);
```

Comment:
minor: consider creating a cancellation token encapsulating a targeted time out & pass it to the call